evm-types.md

```diff
@@ -682,9 +683,9 @@ Addresses
 -   `#lookup` looks up a key in a map and returns 0 if the key doesn't exist, otherwise returning its value.
 
 ```k
-    syntax Int ::= #lookup ( Map , Int ) [function]
+    syntax Int ::= #lookup ( Map , Int ) [function, functional]
  // -----------------------------------------------
-    rule [#lookup.some]: #lookup( (KEY |-> VAL) M, KEY ) => VAL
+    rule [#lookup.some]: #lookup( (KEY |-> {VAL}:>Int) M, KEY ) => VAL
     rule [#lookup.none]: #lookup(               M, KEY ) => 0 requires notBool KEY in_keys(M)
```

Comment:
formatting

Reply:
?

Reply:
Before the change the `=>` were lined up and the `-------` extended to the end of the previous line.